submissions/models.py

```diff
@@ -46,6 +43,12 @@ def __init__(self, *args, **kwargs):
         kwargs['db_index'] = True
         super(AnonymizedUserIDField, self).__init__(*args, **kwargs)
 
+    def deconstruct(self):
+        name, path, args, kwargs = super(AnonymizedUserIDField, self).deconstruct()
+        del kwargs["max_length"]
+        del kwargs["db_index"]
+        return name, path, args, kwargs
+
```

Comment:
@dianakhuang Please review the above changes. We are removing `South` for Django 1.8 upgrade, So as per below links alternative of `add_introspection_rules` is `deconstruct `. 

http://stackoverflow.com/questions/29551367/is-there-a-django-1-7-replacement-for-souths-add-introspection-rules
https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#custom-field-deconstruct-method